tests/datasets/test_loveda.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+from pathlib import Path
+from typing import Generator
+
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.fixtures import SubRequest
+from _pytest.monkeypatch import MonkeyPatch
+
+import torchgeo.datasets.utils
+from torchgeo.datasets import LoveDA
```

Comment:
You'll need to expose LoveDA [here](https://github.com/microsoft/torchgeo/blob/main/torchgeo/datasets/__init__.py) else the import will fail.